domain/src/test/java/org/oppia/domain/util/RatioExtensionsTest.kt

```diff
@@ -5,13 +5,9 @@ import com.google.common.truth.Truth.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.oppia.app.model.RatioExpression
-import org.robolectric.annotation.Config
-import org.robolectric.annotation.LooperMode
 
 /** Tests for [RatioExtensions]. */
 @RunWith(AndroidJUnit4::class)
-@LooperMode(LooperMode.Mode.PAUSED)
```

Comment:
LooperMode paused should be kept--all tests should have this by default.

Reply:
done